doc/tutorial/ranges/views/minimiser/index.md

```diff
@@ -0,0 +1,161 @@
+# Minimiser {#tutorial_minimiser}
+
+[TOC]
+
+This tutorial introduces minimisers. Minimisers are a compact representation of a sequence that is closely related to,
+but more efficient than a representation by k-mers. The minimisers are implemented as a view in SeqAn3. For more
+information about views and how to implement your own view, please see  
+\ref tutorial_ranges and \ref howto_write_a_view .
+
+\tutorial_head{Easy, 20 min, , \ref tutorial_ranges\, \ref howto_write_a_view}
+
+# Motivation
+
+A common way to work with sequences is to obtain their k-mers, but even with a large size of k, the number of k-mers is
+non-trivial, therefore storing k-mers is costly. At the same time, due to the overlap of consecutive k-mers, the
+information they contain is highly redundant. That is why, minimisers come in handy. Minimisers are k-mers, which are
+minimal in a window of a specified size.  Minimal could for example mean lexicographically smallest. By storing only
+these minimal k-mers the storage cost is significantly reduced while maintaining a similar amount of information.
+
+# Minimiser Workflow
+
+Because minimisers are minimal k-mers, they depend on a given k-mer size, a given shape (specifying which positions
+should be considered) and a window size, which has to be greater or equal than the k-mer size. If all these values are
+given, then the minimisers can be obtained by determining all k-mers in the forward and in the backward strand for one
```

Comment:
Here you use backward strand but you did not mention that we are workin on biological sequences yet. I would either mention that in the section title, in the beginning of the tutorial (`In bioinformatics we often represent biological sequences by kmers...`) or before this sentence (`Consider a dna sequence, ...`). (Something a long these lines for example)
Otherwise people might wonder what you mean by backward strand.

Reply:
I added DNA and RNA sequence to the first sentence to make clear I mean biological sequences. :)